### ISSUE TITLE
No focus is set on "Upload random codes" with Talkback on

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#765]
Samsung Galaxy J3 Prime
Android version 7.0

Samsung Galaxy S5
Android version 6.0

App version: 1.0 (53)

With Talkback on, if users press “Cancel” from “Now you can upload your codes” screen, no focus is set on any element on "Upload random codes" screen.

Expected result
The focus should be set on the screen.

Screenshots
![code1](https://user-images.githubusercontent.com/12256067/87825990-1b545900-c846-11ea-881b-e6603d56cc24.gif)
![code4](https://user-images.githubusercontent.com/12256067/87825995-1d1e1c80-c846-11ea-9a85-d88ec8d7b676.gif)

**Comment by mtropper** [comment]
This is caused by the debug menu button. When debug mode is turned off, it works as expected.
